migration/migrating_4_1_4/deploying-cam-4-1-4.adoc

```diff
@@ -19,7 +19,7 @@ After you have installed the {mtc-short} Operator on your clusters, you can laun
 
 == Installing the {mtc-short} Operator
 
-You can install the {mtc-short} Operator with the Operator Lifecycle Manager (OLM) on an {product-title} {product-version} target cluster and on an {product-title} 4.1 source cluster.
+You can install the {mtc-short} Operator with the Operator Lifecycle Manager on an {product-title} {product-version} target cluster and on an {product-title} 4.1 source cluster.
```

Comment:
I noticed that there are still references in the PR to 4.1 specifically. Is this intentional?

Reply:
I have subsequently read your comment:

| Done. That was in a module written by Andrea. She said it was OK to change to attribute.

Can you please update the 4.1 and 4.6 references accordingly?

Reply:
> Is this direct reference to "4.1" intentional, or should it be replaced with `{product-version}`, since we are cherry picking this PR to 4.4, 4.5, 4.6 and 4.7? The same question applies to other references to 4.1 in this PR, which I will not highlight individually in this review.

4.1 is intentional. It's for migrating FROM 4.1. 4.1 has a slightly different UI. This does not affect cherry-picking because the published doc is the version we are migrating TO. So 4.1 > 4.4, 4.1 > 4.5, 4.1 > 4.6. etc.

I updated all the 4.6 references to the product-version attribute. The SOURCE version must stay as it is.

Reply:
Thank you for clarifying that.